app/models/topic.rb

```diff
@@ -4,4 +4,5 @@ class Topic < ApplicationRecord
   belongs_to :user
   has_many :collections_topics
   has_many :collections, through: :collections_topics
+  accepts_nested_attributes_for :collections_topics
```

Comment:
You're not using this. Remove.

Reply:
Hmmm, dont I? I thought this was how I was adding topics to collections 😕

Reply:
Oh maybe you do then. 🤔 Not sure. Try taking it out and see if it breaks. You might be right